client/locations-list-map.js

```diff
@@ -49,6 +49,9 @@ let gSecondaryCluster = null;
 
 let gCurrentViewportCenter = {};
 
+// New object that stores key:value pairs between a filter and its corresponding Selector object
+const filterSelectors = {};
```

Comment:
hmm... I'm not loving more global variables. We were attempting to move away from these.

As I understand it, the goal is to be able to find all selectors with a given filter value and remove it?  If so, can we maybe store the information on the selectr dom nodes itself maybe in a dataset attribute?

Reply:
or maybe that's not possible...sorry, I don't think I understood your code.  I now wonder if this should just be an array of all Selectr objects. Those are basically constant for the duration of the page after initialization I think?

Reply:
Yeah, I'm trying to give other methods like when we click on reset the map the access to the selected selector objects so that's why I declared it glob. I was trying to find a way but I couldn't find a way to go from the Dom element back to the selector objects that generated them. If that's possible, I think we can avoid the global variables.